task.js

```diff
@@ -0,0 +1,248 @@
+// Клас який описує людину
+class Human {
+    constructor(object){
+      this.name = object.name;
+      this.surname = object.surname;
+      this.age = object.age;
+    }
+    getFullName(){
+      return `${this.name} ${this.surname}`;
+    }
+    setFullName(FullName) {
+      [this.name, this.surname] = FullName.split(' ');
+    }
+  }
+  
+  // -------------------------------------Student------------------------------
+  // Клас який успадковує клас Human
```

Comment:
That's pretty nice that you've commented the code out but comments should be in english